version.php

```diff
@@ -0,0 +1,145 @@
+**Note:** Support for <tt>$plugin->supported =</tt> and <tt>$plugin->incompatible =</tt> was added in Moodle 3.9 (MDL-59562).
```

Comment:
Where is all this content file displayed?

Reply:
Ah - this is a mistake and needs removing.